src/main/resources/db/migration/0200/V213__FacilityNumber.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE facilities ADD COLUMN facility_number INTEGER;
+
+WITH facility_numbers AS (
+    SELECT id, RANK() OVER (PARTITION BY organization_id, type_id ORDER BY id) AS facility_number
```

Comment:
Nice, my understanding is the rank + partition criteria produces ranks that are the correct facility numbers.

Reply:
Yeah, `RANK()` returns a 1-indexed sequence number within each partition. A bit more explanation if you're curious: https://www.sqltutorial.org/sql-window-functions/sql-rank/